feature/setting/src/main/java/io/github/droidkaigi/confsched2022/feature/setting/Setting.kt

```diff
@@ -183,7 +187,13 @@ private fun DynamicColorSetting(
 ) {
     Row(
         modifier = modifier
-            .padding(16.dp)
+            .toggleable(
+                value = isDynamicColorEnabled,
+                role = Role.Switch,
+                onValueChange = { onDynamicColorToggle(it) }
+            )
+            .heightIn(min = 56.dp)
+            .padding(horizontal = 16.dp)
```

Comment:
When the vertical padding is set, the height of the item with the Switch does not match the height of the other items, so I removed it.